### DISCUSSION TITLE
automatic switching between iphone and windows

### DISCUSSION THREAD

**Discussion by Tarooc** [steam3d/MagicPods-Windows#77] [Ideas]
Hi, i was wondering if it is possible to add this feature. it would be great if I can switch between iphone and winows 10 pc (like in macbooks). 
thanks

**Comment by steam3d** [comment]
Hi. Unfortunately, it's not possible.

**Reply by Legys** [reply]
A little bit out of the context, but how painful it does when you switch from Apple devices to windows, do I need to pair them each time?

**Reply by steam3d** [reply]
No, AirPods works as regular Bluetooth headphones. it's enough just to pair the AirPods once, and then just connect to it.
